app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -225,6 +200,16 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
         }
     }
 
+
+    private fun updateBookmarkFromTextChanges() {
+        progress_bar_bookmark.visibility = View.VISIBLE
+        val nameText = bookmarkNameEdit.text.toString()
+        val urlText = bookmarkUrlEdit.text.toString()
+        updateBookmarkNode(nameText,urlText)
+    }
+
+
```

Comment:
nit- remove these extra blank lines

Reply:
I think there are still some extra lines here. Should only be one blank line in between the functions